usco/Jam
# UI Redesign and upgrade

As a user I would like a nicer UI for JAM

## TODO:

New /upgraded things: 
- [x] UI for translations
- [x] UI for rotations
- [x] UI for scaling
- [x] UI for editing name & color
- [x] UI for help & settings
- [x] UI for BOM

## How to test
- test ALL elements of UI :
 -  load models and move/rotate/scale them via the new ui and the '3d controls' and check if it all works
 - the above should also work with & without snapping & uniform scaling
 - load a design in view & edit mode and check if all is OK
 - open the bom, add & edit parts and check if all works OK
 - check if progressbar works
